src/pages/auth/JoinPage.tsx

```
@@ -22,88 +20,87 @@ export default function SignupPage() {
-      setCodeError("");
-      // 실제로는 이메일로 코드 발송 API 호출
-      console.log("인증코드 발송:", email);
-    } else {
-      setCodeSent(false);
-      setCodeVerified(false);
-      setCodeError("이메일 형식을 확인해 주세요.");
-      setError("email", {
-        type: "manual",
-        message: "올바르지 않은 형식이에요",
-      });
+  const postSendCode = () => {
```

ESLint

[error] 'postSendCode' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)